.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  pull_request:
+    types: [opened, closed, synchronize]
+  workflow_dispatch:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+
+          # Dependencies to build the deb
+          # sudo apt-get install build-essential devscripts debhelper -y
+
+          # Dependencies for deb-get itself
+          sudo apt-get install apt apt-transport-https bash coreutils curl dpkg grep libc-bin lsb-release sed software-properties-common sudo wget -y
+
+
+      # All these commands should return a 0 status
+      - name: Test zero status
+        run: |
```

Comment:
need to run update first to create /var/cache/deb-get (or ...)
else upstream/main currently fails
see https://github.com/philclifford/deb-get/runs/7484384462?check_suite_focus=true#step:4:8
but https://github.com/philclifford/deb-get/runs/7484487001?check_suite_focus=true looks good